src/default-xpring-client.ts

```diff
@@ -121,7 +122,7 @@ class DefaultXpringClient implements XpringClientDecorator {
     if (!balance) {
       throw new Error(XpringClientErrorMessages.malformedResponse)
     }
-    return BigInt(balance)
+    return bigInt(balance.getDrops())
```

Comment:
why do you need to add getDrops here?

Reply:
balance is an object
```
export namespace XRPDropsAmount {
    export type AsObject = {
        drops: number,
    }
}
```

We have to get the number to make it a bigInt (this should really be a string also).

Reply:
I investigated this. Tyler is definitely right here. What I think is happening is that the balance object has only one field, drops , if you call balance.toString() it just prints the number of drops. 

Presumably BigInt calls this method internally and thus this actually works, despite being semantically incorrect.